### ISSUE TITLE
Sniff errors/warnings not all named

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#297]
Each error/warning can be named:
```php
$phpcsFile->addError($error, $stackPtr, 'SpacingAfter');

//vs the unnamed variant:

$phpcsFile->addError($error, $stackPtr);
```

If they are, they can be selectively disabled:
```xml
<exclude name="WordPress.VIP.RestrictedFunctions.user_meta" />
```

A number of the WP specific sniffs at this moment don't have all errors/warnings named which means that if an errors/warnings is buggy or a project for whatever reason chooses not to use that errors/warnings, they have to disable the complete sniff as the specific errors/warnings can't be disabled.

I'd suggest that naming all errors/warnings generated would be good practice. Opinions ?

**Comment by JDGrimes** [comment]
Duplicate: #140

**Comment by westonruter** [comment]
:+1:

**Comment by jrfnl** [comment]
PR #298 addressed this.

**Comment by westonruter** [comment]
PR merged. Nice work.

**Comment by jrfnl** [comment]
Thanks ;-)
